Mathlib/Data/PNat/Basic.lean

```diff
@@ -19,95 +18,98 @@ import Mathlib.Algebra.Order.Positive.Ring
 /-!
 # The positive natural numbers
 
-This file develops the type `ℕ+` or `pnat`, the subtype of natural numbers that are positive.
-It is defined in `data.pnat.defs`, but most of the development is deferred to here so
-that `data.pnat.defs` can have very few imports.
+This file develops the type `ℕ+` or `PNat`, the subtype of natural numbers that are positive.
+It is defined in `Data.PNat.Defs`, but most of the development is deferred to here so
+that `Data.PNat.Defs` can have very few imports.
 -/
 
-
 deriving instance AddLeftCancelSemigroup, AddRightCancelSemigroup, AddCommSemigroup,
   LinearOrderedCancelCommMonoid, Add, Mul, Distrib for PNat
 
 namespace PNat
 
 @[simp]
-theorem one_add_nat_pred (n : ℕ+) : 1 + n.natPred = n := by
-  rw [nat_pred, add_tsub_cancel_iff_le.mpr <| show 1 ≤ (n : ℕ) from n.2]
-#align pnat.one_add_nat_pred PNat.one_add_nat_pred
+theorem one_add_natPred (n : ℕ+) : 1 + n.natPred = n := by
+  rw [natPred, add_tsub_cancel_iff_le.mpr <| show 1 ≤ (n : ℕ) from n.2]
+#align pnat.one_add_natPred PNat.one_add_natPred
```

Comment:
This looks like a some search-replace went wrong

Reply:
oops O.o